cli/cmd/plugin/ui/web/tanzu-ui/src/components/aws/credential-step/Credentials.scss

```diff
@@ -0,0 +1,45 @@
+.wizard-content-container {
+    .select-label {
+        font-size: 12px;
+        margin-bottom: 10px;
+        font-weight: 600;
+    }
+    .space {
+        padding-right: 50px;
+        font-size: 13px;
+        line-height: 24px;
+        font-weight: 400;
+    }
+    .title {
+        margin-bottom: 25px;
+    }
+    .btn-container {
+        padding-top: 37px;
+    }
+    .select-ms-width {
+        min-width: 143px;
+    }
+    .select-md-width {
+        min-width: 200px;
+    }
+    .control-message-width {
+        width: 200px;
+    }
+    .btn-refresh {
+        margin-top: 5px;
+        font-size: 12px;
+        &:hover {
+            cursor: pointer;
+        }
+    }
+    .vertical-mid {
+        vertical-align: middle;
+    }
+    .blue-text {
```

Comment:
duplicate class, use 'text-blue' which is inherited from index.scss

Reply:
css issues will be handled in another PR.